dockstore-webservice/src/main/java/io/dockstore/webservice/core/Category.java

```diff
@@ -0,0 +1,26 @@
+package io.dockstore.webservice.core;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.v3.oas.annotations.media.Schema;
+import javax.persistence.Entity;
+import javax.persistence.NamedQueries;
+import javax.persistence.NamedQuery;
+
+/**
+ * Describes a Category, which is a dockstore-curated group of entries.
+ * In this implementation, a Category is a Collection under the hood.
+ */
+
+@ApiModel("Category")
+@Schema(name = "Category", description = "Category of entries")
+@Entity
+@NamedQueries({
+        @NamedQuery(name = "io.dockstore.webservice.core.Category.getCategories", query = "SELECT c FROM Category c"),
+        @NamedQuery(name = "io.dockstore.webservice.core.Category.findByName", query = "SELECT c FROM Category c where lower(c.name) = lower(:name)")
+})
+
+public class Category extends Collection {
```

Comment:
Question. are there no collection-specific properties (which don't apply to category) right now? If we plan to add collection-specific properties, how do we do it?

Reply:
That's a drawback to piggybacking Categories on Collections, it's hard to make the concepts diverge.  In the pure notion of a Category, there is no organization, it's just a curated group of entries and some metadata.